chapter_04_uow.asciidoc

```diff
@@ -5,6 +5,20 @@ In this chapter we'll introduce a final piece of the puzzle that really pulls
 the _Repository_ and _Service layer_ patterns together, the _Unit of Work_
 pattern.
 
+A unit of work is a tool that lets us load objects from our persistent
```

Comment:
but i thought a repository was a tool that lets us load objects from permanent storage.

Reply:
A repository is an abstraction over the idea of persistent storage. A Unit of Work is an abstraction over the idea of atomic operation. Is that a better formulation?

Reply:
yes!